src/pages/reference/providers/aws/configuration.mdx

```diff
@@ -7,6 +7,8 @@ export const description = 'Configuring your AWS stacks'
 ```yaml
 # The name of your AWS stack
 name: my-aws-stack-name
+# The provider to use and it's published version (see [Releases](https://github.com/nitrictech/nitric/tags))
```

Comment:
these links don't get converted into actual links.
Just put the url without the markdown link